components/home/news.html

```diff
@@ -1,30 +1,35 @@
+                <h3 class="card-title text-center">Fitness</h3>
+                <p class="card-text text-center">
+                    This is the MAGNET of F3 and it is what attracts other men to join an F3 in the gloomy early mornings.
+
+In F3, leadership of the workout varies from day to day and week to week, meaning that our always-rotating workout leaders are constantly introducing new exercises and ways of doing them. Because participants are challenged and pushed in new ways every day, they achieve an all-around fitness that makes F3 guys supremely fit for endurance challenges, including mud runs, obstacle races, triathlons, marathons and all manner of other adventure challenges.
+                </p>
+            </div>
         </div>
-        <div class="card-footer">
-            <small class="text-muted"></small>Last updated 3 weeks ago</small>
+        <div class="card">
+            <div class="card-body">
+                <h3 class="card-title text-center">Fellowship</h3>
+                <p class="card-text text-center">
+                    This is the GLUE of F3 and it is what makes men stick to F3 through the strong bonds that are formed through our workouts.
+
+There’s something special about pushing through a brutal smokefest with another guy. We encourage those bonds by closing every workout with what we call the COT (Circle of Trust). Every guy gives his birth name, his age and his F3 nickname. Even if you’re brand-new (what we call an FNG), we’ll slap you with an F3 nickname so we can remember who you are the next time you post. We also channel our 2nd F through weekly happy hours and coffeeteria to bond outside of workouts.
+                </p>
+            </div>
         </div>
-    </div>
-    <div class="card mt-3">
-        <div class="card-body text-right">
-            <h5 class="card-title">F3 Austin Region Launch</h5>
-            <h6 class="card-subtitle text-muted">Jan 16th 2021</h6>
-            <p class="card-text mt-3">
-                Integer velit felis, rhoncus rutrum sodales sed, faucibus eget lorem. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Aenean id mi non massa vehicula pellentesque a vitae nibh. Suspendisse arcu felis, tempus a ex id, gravida rhoncus dui. Suspendisse potenti. Vestibulum purus metus, vulputate id feugiat ut, condimentum et ipsum. Lorem ipsum dolor sit amet, consectetur adipiscing elit...
-            </p>
-            <a href="#" class="btn btn-primary">Full Post</a>
-        </div>
-        <div class="card-footer">
-            <small class="text-muted"></small>Last updated 3 weeks ago</small>
+        <div class="card">
+            <div class="card-body">
+                <h3 class="card-title text-center">Faith
+                </h3>
+                <p class="card-text text-center">
+                    This is the DYNAMITE of F3 and it is what gives the men of F3 purpose to lead and give back to our community.
+
```

Comment:
Instead of having blank lines we should create multiple paragraphs with <p></p>

Reply:
I think the blank line came in when I pasted the text.